.travis.yml

```diff
@@ -5,10 +5,10 @@ branches:
 language: node_js
 node_js:
   - "8"
-script:
-  - bash ./src/scripts/deploy.sh
 sudo: required
 install:
   - npm install --global yarn
```

Comment:
I think it's not necessary to install yarn since Travis detects the `yarn.lock` file and does it already,

Reply:
I would also wait til your fixes on linting are merged so you can reorganize the npm scripts to to run the checks. It might also be a good opportunity to add a pre-commit hook so we don't contributors don't find out about them when the CI fails, which is a poor developer experience.

Reply:
Good catch on `yarn.lock` file!

Just to clarify, by pre-commit hooks you mean add linter with `--fix` flag to automatically fix possible style issues?

Reply:
Please do it in another PR, it is better to keep them small.

Reply:
Sure 👍